### PR TITLE
Translate diff

### PR DESCRIPTION
**Pull request by nzw0301**

### CODE REVIEW

File: sources/getting-started/sequential-model-guide.md [hunk 1]
```diff
@@ -90,6 +90,13 @@ final_model.add(Dense(10, activation='softmax'))
 
 <img src="http://s3.amazonaws.com/keras.io/img/two_branches_sequential_model.png" alt="two branch Sequential" style="width: 400px;"/>
 
+このような2つに分岐したモデルは次のようにして学習できる:
+
+```python
+final_model.compile(optimizer='rmsprop', loss='categorical_crossentropy')
+final_model.fit([input_data_1, input_data_2], targets)  # we pass on data array per model input
```
Comment (fchollet) [inline sources/getting-started/sequential-model-guide.md]: There was a typo in the original version: it's "one", not "on"

**Comment by nzw0301** [comment]
Fix it.
